### ISSUE TITLE
Atem logo; bikeshedding thread

### ISSUE THREAD

**Issue by graphicore** [graphicore/Atem-Project#4]
I started this on g+:

https://plus.google.com/u/0/+LasseFister/posts/RGsRev4uAp9

with valuable feedback of @ebensorkin

then:

https://plus.google.com/u/0/+LasseFister/posts/WCNJDUAbA9f

and this valubale feedback by @davelab6:

> I don't like the little wedge triangle on the left side by the pin, its too fussy, better to have the curved ruler part just join the left stem and align with the screw-pin :)
> 
> The M's flat sides get effected by the circular transformation in a bad way, better to have them be angled like normal so that they become flat(ter) in the transform.
> 
> The x strokes cross too low, especially next to that tall P and R
> 
> D should be wider﻿

This is the current version:

![atem-logo](https://cloud.githubusercontent.com/assets/393132/15549457/ea29e61c-22ac-11e6-80bd-ec8b0e30e8d4.png)

(updated to make the compass a bit smaller)

**Comment by graphicore** [comment]
And here is the response to:

> **Dave Crossland**
> If you track them out, I think all the letters could be shorter (more square)﻿
> **Lasse Fister**
> They could be of course. But I kind of like that narrow appearance. I just had a quick test, shorter letters (more squarish), no changed tracking looks pretty good as well. Maybe easier on the eye for most people, but also less distinctive.
> I could open a bike shedding issue on github. Here I have to start a new thread for each new version.﻿
> **Dave Crossland**
> I think more squarish will be way better, because at the moment the text and the icon are vying for top spot, and by making the letters smaller, you clearly define the primary/secondary hierarchy﻿

(fontSize: 20;)

![atem-logo](https://cloud.githubusercontent.com/assets/393132/15547037/754dc908-22a2-11e6-95b0-6399e672352b.png)

(updated to make the compass a bit smaller)

**Comment by graphicore** [comment]
Update incorporating a suggestion of @etunni, Thanks.

![atem-logo](https://cloud.githubusercontent.com/assets/393132/15590617/eabe0102-2398-11e6-8b49-bf4bb08085d7.png)

(updated, made the stroke a bit lighter and the G a bit more open)

**Comment by graphicore** [comment]
The glyphs a bit wider again and a bit less spacing.

![atem-logo](https://cloud.githubusercontent.com/assets/393132/15591316/900d4e9e-239c-11e6-9a47-dbc14a50e891.png)

**Comment by impallari** [comment]
Some letters are wide, other are narrow.

Here is a guide to sans uppercase proportions:
15 units = I
22 units = J
29 units = L
30 units = F
31 units = S
32 units = E, T
33 units = P, Y, Z
35 units = B, K, R, X
36 units = C, V
37 units = A
39 units = D, G, U
40 units = H, N
42 units = O, Q
48 units = M
53 units = W

How to calculate your unit value:
Lets say your /H advance-width (sidebearings + contours) is 600 points.
600 / 40 = 15 points per unit.
Now you can use that value to calculate the advance-width of all the other uppercase letters.

**Comment by graphicore** [comment]
@impallari  I'll give it a try!

Actually, there are no points, this is all just "units" in a way. So, I don't get where the 600 and 40 is coming from ;-)

The units here are from 0 to 1 can be bigger or smaller of course, but 1 is the base. The `E` for example has:

``` yaml
        width: .9
        before: .1
        after: .1
```

What would you suggest as the base character? The one that defines the 1.

**Comment by impallari** [comment]
You can use any base glyph you like to calculate the other glyphs.

For example, using the /E that you just posted:
Your /E advance-width is .9+.1+.1 = 1.1
In my guide the /E is 32 units, so each "proportion unit" will be 1.1 / 32 = 0.034375

Now, if you want to calculate your full /H advance width, just do 0.034375 x 40 = 1.375
And you can distribute it something like:
width: .775
before: .3
after: .3

**Comment by graphicore** [comment]
Yeah, sure I can, but which glyph should be the ideal standard "1" also, how do you distribute space vs form. I.e. how to get to 0.1 for the space before and after, it seems arbitrary to me and the usual approach is to trail and error, or?

**Comment by impallari** [comment]
Typically the /H and /O are used as reference, but we don't have /H here, so I will suggest to use the /O.

Or use any other letter you like. However, since your current /A is very wide, if you use it as a reference all the alphabet will turn out wider. In a similar way, your current /M is narrow, if you use it as a reference all the resulting alphabet will turn out to be narrow.

For before and after:
Typically you use more space in the /H, lets say .3
A little less in the circular ones like the /O, lets say .2
and a minimal value in the diagonals (/A /V), lets say .1

But here you /O is not circular, so i will use the same value as in the /H (.3) for it.

A quick guide, to initiate the trial and error process :)
.1 A T V X .1
.3 D E G R .2
.3 L P .1
.3 I M N O .3
.2 S .2

**Comment by graphicore** [comment]
> Or use any other letter you like. However, since your current /A is very wide, if you use it as a reference all the alphabet will turn out wider. In a similar way, your current /M is narrow, if you use it as a reference all the resulting alphabet will turn out to be narrow.

Yeah, I can make all the letters wider and narrower by the stroke of a key. One unit of 1 refers to how many units fit in a circle. So, I can say my circle fits 50 units and that will happen. It's all relative of course, but it's easier to think about it if there's a reasonable default (for 1).

**Comment by impallari** [comment]
Now I want to see how it look :)
